doc/services/storage/zms/zms.rst

```diff
@@ -179,88 +180,68 @@ counter as the empty ATE.
 When closing a sector, all the remaining space that has not been used is filled with garbage data
 to avoid having old ATEs with a valid cycle counter.
 
-Triggering Garbage collection
+Triggering garbage collection
 =============================
 
 Some applications need to make sure that storage writes have a maximum defined latency.
-When calling a ZMS write, the current sector could be almost full and we need to trigger the GC
-to switch to the next sector.
+When calling ZMS to make a write, the current sector could be almost full so that it needs to
+trigger the GC to switch to the next sector.
 This operation is time consuming and it will cause some applications to not meet their real time
 constraints.
 ZMS adds an API for the application to get the current remaining free space in a sector.
-The application could then decide when needed to switch to the next sector if the current one is
-almost full and of course it will trigger the garbage collection on the next sector.
+The application could then decide when to switch to the next sector if the current one is almost
+full. This will of course trigger the garbage collection on the next sector.
```

Comment:
```suggestion
full. This will of course trigger garbage collection on the next sector.
```
or
```suggestion
full. This will of course trigger the garbage collection operation on the next sector.
```